.github/workflows/ci.yml

```diff
@@ -425,14 +424,8 @@ jobs:
           ref: ${{ needs.build-info.outputs.sha }}
       - name: Download compiled Ozone binaries
         uses: actions/download-artifact@v4
-        if: ${{ matrix.suite }} == 'MR' # until HDDS-11053 is fixed
-        with:
-          name: ozone-bin-${{ env.BUILD_JAVA_VERSION }}
-      - name: Download compiled Ozone binaries
-        uses: actions/download-artifact@v4
-        if: ${{ matrix.suite }} != 'MR'
         with:
-          name: ozone-bin-${{ env.TEST_JAVA_VERSION }}
+          name: ozone-bin
```

Comment:
Are Ozone binaries built by Java17?  If so, are they fixed to fun MR with Java17 now?